sdcm/cluster.py

```diff
@@ -1891,6 +1897,22 @@ def upgrade_manager_agent(self, scylla_mgmt_repo, start_agent_after_upgrade=True
             else:
                 self.remoter.run("sudo systemctl start scylla-manager-agent")
 
+    def clean_scylla_data(self):
+        """Clean all scylla data file
+
+        Commands are taken from instruction in docs.
+        See https://docs.scylladb.com/operating-scylla/procedures/cluster-management/clear_data/
+        """
+        clean_commands_list = [
+            "sudo rm -rf /var/lib/scylla/data",
+            "sudo find /var/lib/scylla/commitlog -type f -delete",
+            "sudo find /var/lib/scylla/hints -type f -delete",
+            "sudo find /var/lib/scylla/view_hints -type f -delete"
```

Comment:
This assumes a fixed structure of directories.

What if the cluster's config file decides on a different structure?

Reply:
@bentsi does sct support and use changing structure of directories?

Reply:
we don't support running Scylla in the directory other than the default